.github/workflows/ci.yml

```diff
@@ -10,6 +10,59 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  hub:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare
+        run: |
+          echo "BAKE_TARGET=image-cross" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REPO_SLUG }}
+          ### frontend versioning
+          ### on semver tag:
+          # second-state/runwasi:1.2.3
+          # second-state/runwasi:1.2
+          # second-state/runwasi:1
+          # second-state/runwasi:latest
+          ### on pre-release tag:
+          # second-state/runwasi:1.1.0-rc.1
+          ### on push default branch (main):
+          # second-state/runwasi:main
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+            type=ref,event=branch
+            type=raw,value=latest
+            type=ref,event=pr
+          bake-target: meta-helper
+          flavor: |
+            latest=false
+      - name: Login to DockerHub
+        if: ${{ github.event_name != 'pull_request' }}
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
```

Comment:
With this github action an admin of the repository should add these two secrets so that we can push the resulting image

Reply:
These two secrets are added.